scipy/stats/tests/test_qmc.py

```diff
@@ -209,39 +212,57 @@ def test_update_discrepancy(self):
                                              r"broadcastable"):
             update_discrepancy(x_new, space_1[:-1], disc_init)
 
+    def test_perm_discrepancy(self):
+        seed = np.random.RandomState(123456)
+        qmc_gen = qmc.LatinHypercube(5, seed=seed)
+        sample = qmc_gen.random(10)
+        disc = qmc.discrepancy(sample)
+
+        for i in range(100):
+            row_1 = np.random.randint(10)
+            row_2 = np.random.randint(10)
+            col = np.random.randint(5)
+
+            disc = _perturb_discrepancy(sample, row_1, row_2, col, disc)
+            sample[row_1, col], sample[row_2, col] = (
+                sample[row_2, col], sample[row_1, col])
+            disc_reference = qmc.discrepancy(sample)
+            assert_allclose(disc, disc_reference)
+
     def test_discrepancy_alternative_implementation(self):
         """Alternative definitions from Matt Haberland."""
+
         def disc_c2(x):
             n, s = x.shape
             xij = x
             disc1 = np.sum(np.prod((1
-                                    + 1/2*np.abs(xij-0.5)
-                                    - 1/2*np.abs(xij-0.5)**2), axis=1))
+                                    + 1 / 2 * np.abs(xij - 0.5)
+                                    - 1 / 2 * np.abs(xij - 0.5) ** 2), axis=1))
```

Comment:
Let's not do this in this PR. They are subjective, and they are unrelated to rest of the PR.

Reply:
Ok sure I will remove all these.